extensions/extensions-helm/extensions-helm-spark-infrastructure/aissemble-spark-history-chart/templates/deployment.yaml

```diff
@@ -2,21 +2,23 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ .Release.Name }}-shs
+  name: "{{ .Values.app.name | default .Chart.Name }}"
```

Comment:
We've seen issues with this when projects upgrade from 1.8 to 1.9.  We were going to just update the instructions for migrating to the new umbrella chart to include updating the Tiltfile with a proper release name, but it might make sense to backport these changes instead.

Reply:
I think if we don't back port this, it might make sense to use `default .Release.Name` just so the change is less jarring going from 1.9 to 1.10

Reply:
Going to backport these changes so no need to default to Release.name.